conda_build/build.py

```diff
@@ -1846,16 +1825,6 @@ def test(recipedir_or_package_or_metadata, config, stats, move_broken=True):
     utils.rm_rf(metadata.config.test_dir)
     _extract_test_files_from_package(metadata)
 
-    # When testing a .tar.bz2 in the pkgs dir, clean_pkg_cache() will remove it.
-    # Prevent this. When https://github.com/conda/conda/issues/5708 gets fixed
-    # I think we can remove this call to clean_pkg_cache().
-    in_pkg_cache = (not hasattr(recipedir_or_package_or_metadata, 'config') and
```

Comment:
This provides a nice speedup, especially on conda 4.5, but unfortunately, it's still not well behaved in conda (tested on 4.5.5).  This bit needs to stay.